changelog.txt

```diff
@@ -6,7 +6,7 @@ The **synced_collections** package follows `semantic versioning <https://semver.
 
 
 [0.0.0] -- XXXX-XX-XX
----------------------
+=====================
```

Comment:
Just FYI, to answer https://github.com/glotzerlab/synced_collections/pull/7#discussion_r1189909613, I removed the overall version headings to make the automated parsing of the changelog for the release easier. Don't think the headers add much anyway.